docs/src/rules/no-extra-parens.md

```diff
@@ -130,6 +131,24 @@ for (;(a = b););
 
 :::
 
+### ternaryOperandBinaryExpressions
+
+Examples of **correct** code for this rule with the `"all"` and `{ "ternaryOperandBinaryExpressions": false }` options:
+
+```js
+/* eslint no-extra-parens: ["error", "all", { "ternaryOperandBinaryExpressions": false }] */
+
+(a && b) ? foo : bar
```

Comment:
```suggestion
(a && b) ? foo : bar;
```

To make sure that each line is a separate example. Without `;`, this expression would be continued with `(a - b > a)...` on line 143.